models/auth/token_scope.go

```diff
@@ -269,7 +382,7 @@ func (bitmap AccessTokenScopeBitmap) ToScope() AccessTokenScope {
 	scope := AccessTokenScope(strings.Join(scopes, ","))
 	scope = AccessTokenScope(strings.ReplaceAll(
 		string(scope),
-		"repo,admin:org,admin:public_key,admin:org_hook,admin:user_hook,notification,user,delete_repo,package,admin:gpg_key,admin:application",
+		"delete:activitypub,delete:admin,delete:misc,delete:notification,delete:organization,delete:package,delete:issue,delete:repository,delete:user",
```

Comment:
`strings.ReplaceAll(scope, "every is delete", "all")` :

Why "delete" scope means "all"? I guess there is an assumption that "delete" also means "write"+"read", but it is not  a strong assumption.